<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>toSyntaxSetupUI</class>
 <widget class="QWidget" name="toSyntaxSetupUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>631</width>
    <height>572</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form1</string>
  </property>
  <property name="toolTip">
   <string/>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_7">
   <property name="spacing">
    <number>6</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="OptionsTab">
      <attribute name="title">
       <string>Options</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_9">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Syntax highlighting</string>
           </property>
           <property name="buddy">
            <cstring>SyntaxHighlightingInt</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="SyntaxHighlightingInt">
           <item>
            <property name="text">
             <string extracomment="Use Scintilla native SQL Lexer">QsciSQL</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string extracomment="Use vendor specific SQL Lexer (if implemented)">Native</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_10">
         <item>
          <widget class="QLabel" name="EditorTypeLabel">
           <property name="text">
            <string>Editor Type</string>
           </property>
           <property name="buddy">
            <cstring>EditorTypeInt</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="EditorTypeInt">
           <item>
            <property name="text">
             <string>Scintilla</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Em</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QCheckBox" name="UseMaxTextWidthMarkBool">
           <property name="toolTip">
            <string>Display the line/column marker at the specified position</string>
           </property>
           <property name="text">
            <string>Use &amp;maximum text width mark</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="MaxTextWidthMarkInt">
           <property name="toolTip">
            <string>Specify the column (text width) where to show marker</string>
           </property>
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>999999</number>
           </property>
           <property name="value">
            <number>80</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QCheckBox" name="CodeCompleteBool">
         <property name="toolTip">
          <string>Activate code completion in SQL editors. Not recommended on slow connections.</string>
         </property>
         <property name="text">
          <string>&amp;Code completion</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="KeywordUpperBool">
         <property name="toolTip">
          <string>Convert all keywords to uppercase when displayingsyntaxhighlighted text.  Only available for monospaced fonts.</string>
         </property>
         <property name="text">
          <string>&amp;Keyword upper</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="ObjectNamesUpperBool">
         <property name="toolTip">
          <string>Table, view, column names etc. are extracted in upper-case</string>
         </property>
         <property name="text">
          <string>Object names upper</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="CompleteSortBool">
         <property name="toolTip">
          <string>Sort the available completion rows alphabetically or in the order the are.</string>
         </property>
         <property name="text">
          <string>&amp;Sort completion rows</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QCheckBox" name="UseEditorShortcutsBool">
           <property name="text">
            <string>&amp;Editor shortcuts (TAB replacement)</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="EditorShortcutsEdit">
           <property name="text">
            <string>E&amp;dit...</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QCheckBox" name="AutoIndentBool">
         <property name="toolTip">
          <string>Indent a new line in an editor to the same column as the previous line</string>
         </property>
         <property name="text">
          <string>&amp;Indent same as previous line</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="UseSpacesForIndentBool">
         <property name="toolTip">
          <string>If set the editor will insert spaces rather than tabs</string>
         </property>
         <property name="text">
          <string>Use only spaces when indenting</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QLabel" name="TabStopLabel">
           <property name="toolTip">
            <string>Number of character width of tabstops</string>
           </property>
           <property name="whatsThis">
            <string>Number of character width of tabstops</string>
           </property>
           <property name="text">
            <string>Tab stop width</string>
           </property>
           <property name="wordWrap">
            <bool>false</bool>
           </property>
           <property name="buddy">
            <cstring>TabStopInt</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="TabStopInt">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="value">
            <number>8</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="TextExampleFont">
           <property name="text">
            <string>Text example</string>
           </property>
           <property name="wordWrap">
            <bool>false</bool>
           </property>
           <property name="buddy">
            <cstring>TextExampleChoose</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="TextExampleChoose">
           <property name="text">
            <string>Choose</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="CodeExampleFont">
           <property name="text">
            <string>Code example</string>
           </property>
           <property name="wordWrap">
            <bool>false</bool>
           </property>
           <property name="buddy">
            <cstring>CodeExampleChoose</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="CodeExampleChoose">
           <property name="text">
            <string>Choose</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="ResultExampleFont">
           <property name="text">
            <string>List example</string>
           </property>
           <property name="wordWrap">
            <bool>false</bool>
           </property>
           <property name="buddy">
            <cstring>ResultExampleChoose</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="ResultExampleChoose">
           <property name="text">
            <string>Choose</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="TextLabel1">
         <property name="text">
          <string>Filename extensions</string>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
         <property name="buddy">
          <cstring>Extensions</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="Extensions"/>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>93</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="SyntaxTab">
      <attribute name="title">
       <string>Syntax</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_8" stretch="1,2">
       <item>
        <widget class="QGroupBox" name="GroupBox2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Sy&amp;ntax components</string>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0" colspan="3">
           <widget class="QListWidget" name="SyntaxComponent"/>
          </item>
          <item row="1" column="0" colspan="2">
           <widget class="QCheckBox" name="CaretLineBool">
            <property name="text">
             <string>Caret line (Alpha)</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0" colspan="2">
           <widget class="QSlider" name="CaretLineAlphaInt">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QPushButton" name="PickCaretButton">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>50</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>&amp;Pick</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="FGColorLabel">
            <property name="text">
             <string>Foreground</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLabel" name="FGSample">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
              <horstretch>1</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>75</width>
              <height>0</height>
             </size>
            </property>
            <property name="autoFillBackground">
             <bool>true</bool>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="3" column="2">
           <widget class="QPushButton" name="PickFGButton">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>50</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>&amp;Pick</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="BGColorLabel">
            <property name="text">
             <string>Background</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QLabel" name="BGSample">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
              <horstretch>1</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>75</width>
              <height>0</height>
             </size>
            </property>
            <property name="autoFillBackground">
             <bool>true</bool>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="4" column="2">
           <widget class="QPushButton" name="PickBGButton">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>50</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>Pick</string>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="FontLabel">
            <property name="text">
             <string>Font</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QLabel" name="FontSample">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
              <horstretch>1</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>75</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string> Sample </string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="5" column="2">
           <widget class="QPushButton" name="PickFontButton">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>50</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>Pick</string>
            </property>
            <property name="autoDefault">
             <bool>false</bool>
            </property>
            <property name="default">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="toSqlText" name="Example" native="true">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>0</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="IndentTab">
      <attribute name="title">
       <string>Indent</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_11">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Indent options</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_12">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Line width:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="IndentLineWidthInt">
              <property name="minimum">
               <number>60</number>
              </property>
              <property name="maximum">
               <number>300</number>
              </property>
              <property name="value">
               <number>180</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_13">
            <item>
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>Indent depth:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="IndentWidthInt">
              <property name="maximum">
               <number>10</number>
              </property>
              <property name="value">
               <number>3</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="Line" name="line">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="ReUseNewlinesBool">
            <property name="text">
             <string>Re-use new lines</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="LinebreakLabel">
            <property name="text">
             <string>Line break for:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="BreakOnSelectBool">
            <property name="text">
             <string>SELECT</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="BreakOnFromBool">
            <property name="text">
             <string>FROM</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="BreakOnWhereBool">
            <property name="text">
             <string>WHERE</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="BreakOnGroupBool">
            <property name="text">
             <string>GROUP BY / HAVING</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="BreakOnOrderBool">
            <property name="text">
             <string>ORDER BY</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="BreakOnModelBool">
            <property name="text">
             <string>MODEL</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="BreakOnPivotBool">
            <property name="text">
             <string>PIVOT</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="Line" name="line_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="WidthModeBool">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Width mode</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="toSqlText" name="IndentExample" native="true">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>0</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>toSqlText</class>
   <extends>QWidget</extends>
   <header>editor/tosqltext.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>SyntaxHighlightingInt</tabstop>
  <tabstop>EditorTypeInt</tabstop>
  <tabstop>MaxTextWidthMarkInt</tabstop>
  <tabstop>UseMaxTextWidthMarkBool</tabstop>
  <tabstop>CodeCompleteBool</tabstop>
  <tabstop>KeywordUpperBool</tabstop>
  <tabstop>ObjectNamesUpperBool</tabstop>
  <tabstop>CompleteSortBool</tabstop>
  <tabstop>UseEditorShortcutsBool</tabstop>
  <tabstop>EditorShortcutsEdit</tabstop>
  <tabstop>AutoIndentBool</tabstop>
  <tabstop>UseSpacesForIndentBool</tabstop>
  <tabstop>TabStopInt</tabstop>
  <tabstop>TextExampleChoose</tabstop>
  <tabstop>PickFontButton</tabstop>
  <tabstop>CodeExampleChoose</tabstop>
  <tabstop>ResultExampleChoose</tabstop>
  <tabstop>Extensions</tabstop>
  <tabstop>SyntaxComponent</tabstop>
  <tabstop>PickFGButton</tabstop>
  <tabstop>PickBGButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>CodeCompleteBool</sender>
   <signal>toggled(bool)</signal>
   <receiver>CompleteSortBool</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
